ci/ctests/scripts/setup.sh.in

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -ux
+
+TEST_NAME=${1:?"Name of the test is required"}
+YAML_FILE=${2:?"Name of the CI yaml file for the test"}
+
+# CMake to fill these variables
+HOMEgfs="@GW_BASELINE_DIR@"
```

Comment:
This should be:
```suggestion
HOMEgfs="@PROJECT_SOURCE_DIR@"
```
Ditto for the other `.in` scripts in this directory

Reply:
No, the only way to have **HOMEgfs** set to `@PROJECT_SOURCE_DIR@` in this way is to have` CMakeLists.txt` at the top of the repo tree.  When the `project()` call is made it is set and in immutable.  I will fall back on how I **HOMEgfs** to this value in the first place.

Reply:
That is correct.  The top-level CMakelists.txt should be in the top of the project, and not here.